mail_statistics_extras/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==========================
+Mail statistics extra info
+==========================
+
+This module add extra info to mail statictics records:
+
+* email_from
+* email_to
+* subject
+
+In order to give a more useful information to final user when listing email
+statistics related with a mass mailing message
+
+
+Usage
+=====
```

Comment:
- Go to Marketing/ Mass Mailing
- Create 
- Set a Subject
- Send the email
- Check the sent email and open it
- Go to Smart Button /number of emails
- check now you can see subject; email_from; email_to; sent date, opened date